boards/shields/mikroe_silent_step_2_click/mikroe_silent_step_2_click.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Navimatix GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mikrobus_i2c {
+	status = "okay";
+	pca9538a_mikroe_silent_step_2_click: pca9538a@70 {
+		status = "okay";
+		compatible = "nxp,pca9538";
+
+		reg = <0x70>;
+
+		gpio-controller;
+		ngpios = <8>;
+		#gpio-cells = <2>;
+
+		gpio-reserved-ranges = <3 1>;
```

Comment:
Setting gpio-reserved-ranges on nxp,pca9538 has no effect - the driver does not handle this.